footprints/main/tests/__init__.py

```diff
@@ -1,6 +1,7 @@
 # flake8: noqa
 from test_admin import *
```

Comment:
You shouldn't need to import stuff in `tests/__init__.py` anymore. Newer django/nose/jenkins setups find the test files without it.